app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -220,3 +220,8 @@ export default {
   }
 };
 </script>
+
+<style lang="scss" scoped>
+@import '~bootstrap/dist/css/bootstrap.min.css';
+@import 'https://unpkg.com/formiojs@4.11.2/dist/formio.builder.min.css';
+</style>
```

Comment:
to read our custom css rules after formio's we should put them here. 
i've put them in the main styles.scss to keep things organised but i dont believe they have a higher specificity when there in that file.